doc/nrf/gs_modifying.rst

```diff
@@ -95,15 +122,80 @@ Typically, the :file:`CMakeLists.txt` files include at least the :file:`main.c`
 Configuring your application
 ****************************
 
-If your application uses a provided library or targets a specific board, you might want to change the default configuration of the library or board.
+You might want to change the default options of the application.
 There are different ways of doing this, but not all will store your configuration permanently.
 
-The default configuration for a library is specified in its :file:`Kconfig` file.
-Similarly, the default configuration for a board is specified in its :file:`*_defconfig` file (and its :file:`Kconfig.defconfig` file, see :ref:`zephyr:default_board_configuration` in the Zephyr documentation for more information).
-The configuration for your application, which might override some default options of the libraries or the board, is specified in a :file:`prj.conf` file in the application directory.
+Zephyr's configuration system is distributed, and so is the |NCS| one.
+This means that there are multiple sources of configuration.
+In the |NCS|, the configuration system consists of the following elements:
+
+* Kconfig configuration files that focus on software-related options.
+* Devicetree specification (DTS) configuration files that focus on hardware-related options.
+* Partition manager configuration files that focus on memory layout.
+
```

Comment:
We should provide links to where the user can learn more about each configuration system.

Reply:
Kconfig: https://docs.zephyrproject.org/latest/guides/build/index.html#configuration-system-kconfig
DT: https://docs.zephyrproject.org/latest/guides/dts/index.html#dt-guide
PM: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/scripts/partition_manager/partition_manager.html

Reply:
Added links to Kconfig and DT.